custom/icds_reports/sqldata.py

```diff
@@ -511,13 +523,13 @@ def columns(self):
         ]
 
 
-class AggCCSRecordMonthlyDataSource(SqlData):
+class AggCCSRecordMonthlyDataSource(ProgressReportSqlData):
     table_name = 'agg_ccs_record_monthly'
     engine_id = 'icds-test-ucr'
 
     def __init__(self, config=None, loc_level='state', show_test=False):
         super(AggCCSRecordMonthlyDataSource, self).__init__(config)
-        self.loc_key = '%s_site_code' % loc_level
+        self.loc_key = '%s_id' % loc_level
```

Comment:
Is it safe to change this? Why was it `site_code` before? I thought we had changed all of these to _id

Reply:
yep, it seems Lukasz forgot to change this. Report is not working correctly due to using key site_code instead of id

Reply:
Thanks. @lwyszomi can you verify this?